### ISSUE TITLE
[feature request] Make system emails translatable

### ISSUE THREAD

**Issue by WebShapedBiz** [Librarika/Issues#690]
Hi.

I've just learned that the system emails are not translatable, so all new members, for example, will get an English invitation email where they should confirm their membership. That kinda puts a question mark on the whole translation feature, doesn't it? I mean, there is no point in partial translation, you either have it or you don't.

Cheers.

**Comment by gekaki** [comment]
I agree, more items need to be translatable. It makes it harder to read in different languages.
